README.md

```diff
@@ -21,10 +21,55 @@ Run `make build`.
 Run `make test` to run all unit and integration tests. To run the integration test, set the ClickHouse database URL to
 the environment variables as an `CLICKHOUSE_URL`.
 
+## Source
+
+The ClickHouse Source Connector allows you to move data from the ClickHouse table to Conduit Destination connectors.
+
+It supports all engines of the [MergeTree](https://clickhouse.com/docs/en/engines/table-engines/#mergetree)
+and [Log](https://clickhouse.com/docs/en/engines/table-engines/#log) families.
+
+The iterator selects existing rows from the selected table in batches with ordering and where claus:
+
+```
+SELECT {{config.columns}}
+FROM {{config.table}}
+WHERE {{config.orderingColumn}} > {{position}}
+ORDER BY {{config.orderingColumn}};
+```
+
+### CDC
+
+When all existing rows have been read, the iterator will start selecting all newly inserted rows. CDC does not support
+the capture of updated or deleted operation. All records have an `OperationCreate` operation.
+
+### Position
+
+The position contains the `orderingColumn` value of the last processed row. This value is used in the where clause of
+the SELECT query.
```

Comment:
Since this README is more about the usage and getting started with the connector, I think this comment is more for the code.

Reply:
Are you talking about the whole paragraph `Position`?